From 75583cf220b2dc7a0f2086b7f233c76a87fc203e Mon Sep 17 00:00:00 2001
From: Tomi Ollila <tomi.ollila@iki.fi>
Date: Sun, 20 Oct 2024 19:28:48 +0300
Subject: [PATCH 3/7] fixed and moved braceright from ']' to '}'

---
 src/accelUtil.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/accelUtil.cpp b/src/accelUtil.cpp
index d0fc106..1ff1a94 100644
--- a/src/accelUtil.cpp
+++ b/src/accelUtil.cpp
@@ -73,7 +73,6 @@ const KeyPair keycodes[] = {
    { "@", Qt::Key_At },
    { "[", Qt::Key_BracketLeft },
    { "\\", Qt::Key_Backslash },
-   { "]", Qt::Key_BraceRight },
    { "]", Qt::Key_BracketRight },
    { "_", Qt::Key_Underscore },
    { "`", Qt::Key_QuoteLeft },
@@ -197,6 +196,7 @@ const KeyPair keycodes[] = {
    { "z", Qt::Key_Z },
    { "{", Qt::Key_BraceLeft },
    { "|", Qt::Key_Bar },
+   { "}", Qt::Key_BraceRight },
    { "~", Qt::Key_AsciiTilde },
 };
 
-- 
2.49.0

